CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Add filter to `MERGE` clauses to reduce the number of scanned records.
+
```

Comment:
Maybe make it `Changed` since DML queries now consists of two statements, which is a breaking change.

```suggestion
### Changed
- Add filter to `MERGE` clauses to reduce the number of scanned records. DML queries now consist of two statements, a `SET` and a `MERGE`, instead of just a `MERGE` query.

```

Reply:
Fair enough! Commited the suggestion